tests/testthat/test-REST.R

```diff
@@ -38,3 +38,27 @@ test_that("generic get w/ and w/o auth", {
 	
 })
 
+test_that("REST_helpers tests", {
+	item <- sbtools_GET("https://www.sciencebase.gov/catalog/item/5c4f4a04e4b0708288f78e07")
+
+	expect_error(sbtools_GET("https://www.sciencebase.gov/catalog/item/a04e4b0708288f78e07"), 
+							 "Invalid Item ID")
+	
+	expect_error(sbtools_GET("https://www.sciencebase.gov/catalog/item/a04e4b0708288f78e07", session = list(session = "borked")),
+							 "Session is not valid.")
+	
+	session <- httr::handle("https://google.com")
+	attributes(session) <- c(attributes(session), list(birthdate=Sys.time()))
+	
+	expect_warning(sbtools_GET("https://www.sciencebase.go/catalog/item/5c4f4a04e4b0708288f78e07", session = NULL))
+	
+	put_test <- sbtools_PUT("https://www.sciencebase.gov/catalog/item/5c4f4a04e4b0708288f78e07", "test", session = session)
+	
+	expect_equal(put_test$status_code, 405)
+	
+	delete_test <- sbtools_DELETE("https://www.sciencebase.gov/catalog/item/5c4f4a04e4b0708288f78e07", session = session)
+
+	expect_equal(delete_test$status_code, 405)
```

Comment:
So with the 405 expected, these are just testing that the package has at least tried the put/delete?

Reply:
Yeah -- I'm just trying to get some lines covered so we have something. More testing coming later that will only run in local dev when logged in that will do more here.